brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post Grid - bottom spacing and hover color issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/64be0980-24b0-4fb2-9114-60beee495cfe?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop post grid.
- Add styling to Read more button (color, hover, spacing and border)
- Check is it visible in editor as well as on frontend.

### Checklist:
